tests/test_unsafe.py

```diff
@@ -314,13 +314,14 @@ def test_update(
         run_update(dst, overwrite=True, unsafe=unsafe)
 
 
-@pytest.mark.parametrize("unsafe", [False, True])
+@pytest.mark.parametrize("unsafe", [False, "--trust", "--UNSAFE"])
 def test_update_cli(
     tmp_path_factory: pytest.TempPathFactory,
     capsys: pytest.CaptureFixture[str],
     unsafe: bool,
```

Comment:
This type hint needs an update now:

```suggestion
    unsafe: Union[bool, str],
```

And `Union` needs to be imported.

Reply:
How do you feel about postponed annotations @sisp @yajo? Allowing to do `bool | str`.

Reply:
I find `bool | str` more readable, and it's shorter. Is there any disadvantage because of the `__future__` import?

Reply:
As long as you don't declare Pydantic models inside of functions, there will be no issues (both with or without `__future__` annotations).

Reply:
I'd vote +1 for converting the code base to annotations, we should do a batch conversion in a separate PR though.

WDYT, @yajo?